core/src/main/kotlin/cz/ackee/localizer/plugin/core/LoadLocalizationUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package cz.ackee.localizer.plugin.core
+
+import cz.ackee.localizer.plugin.core.auth.CredentialsService
+import cz.ackee.localizer.plugin.core.auth.CredentialsServiceImpl
+import cz.ackee.localizer.plugin.core.configuration.ConfigurationParser
+import cz.ackee.localizer.plugin.core.configuration.ConfigurationParserImpl
+import cz.ackee.localizer.plugin.core.localization.LocalizationsRepository
+import cz.ackee.localizer.plugin.core.localization.LocalizationsRepositoryImpl
+import cz.ackee.localizer.plugin.core.sheet.XmlGenerator
+
+interface LoadLocalizationUseCase {
```

Comment:
i'd vote for removing the interface and just use a class since it does not add any value, but im not sure what's the current policy in Ackee about this :D

Reply:
Fixed in bc7356862a68350b3fe58e81dc7c1e497d587bf8.